### PR TITLE
sdk: add control to disable ccbm in camera

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

File: sdk/include/aditof/camera.h [hunk 1]
```diff
@@ -493,6 +493,13 @@ class SDK_API Camera {
      */
     virtual Status adsd3500GetStatus(int &chipStatus, int &imagerStatus) = 0;
 
+    /**
+     * @brief Enable/disable ccb as master 
+     * @param[out] enable - 1 - disable, 0 - enable
+     * @return Status
+     */
+    virtual Status adsd3500disableCCBM(bool disable) = 0;
```
Comment (dNechita) [inline sdk/include/aditof/camera.h:501]: For consistency, you should use capital D (adsd3500d -> adsd3500D)

File: sdk/include/aditof/camera.h [hunk 2]
```diff
@@ -493,6 +493,13 @@ class SDK_API Camera {
      */
     virtual Status adsd3500GetStatus(int &chipStatus, int &imagerStatus) = 0;
 
+    /**
+     * @brief Enable/disable ccb as master 
+     * @param[out] enable - 1 - disable, 0 - enable
```
Comment (dNechita) [inline sdk/include/aditof/camera.h:498]: param[in]

**Comment by dNechita** [comment]
Python bindings need to be updated as well.

File: sdk/include/aditof/camera.h (hunk 1 above)
Comment (SeptimiuVana) [reply sdk/include/aditof/camera.h]: Fixed.

File: sdk/include/aditof/camera.h (hunk 2 above)
Comment (SeptimiuVana) [reply sdk/include/aditof/camera.h]: Fixed.

**Review by dNechita — APPROVED** [review]

**Pull request merged**